tidb-cloud/set-up-public-endpoint-connections-serverless.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Connect to TiDB Cloud Serverless via Public Endpoint
+summary: Learn how to configure and manage public access to your TiDB Cloud Serverless cluster securely.
+---
+
+# Connect to TiDB Cloud Serverless via Public Endpoint
+
+This article describes the public connectivity option for TiDB Cloud Serverless. You will learn key concepts for securely managing a TiDB Cloud Serverless cluster accessible via the internet.
+
+> **Note:**
+>
+> This document applies to **TiDB Cloud Serverless**. For instructions on configuring an IP access list for **TiDB Cloud Dedicated**, see [Configure an IP Access List for TiDB Cloud Dedicated](/tidb-cloud/configure-ip-access-list.md).
+
+## Public endpoints
+
+Configuring public access on your TiDB Cloud Serverless cluster allows the cluster access through a public endpoint. That is, the cluster is accessible through the internet. The public endpoint is a publicly resolvable DNS address. The term "authorized network" refers to a range of IP addresses you choose to permit access to your cluster. These permissions are enforced through **firewall rules**.
+
+### Characteristics of public access
+
+- Only specified IP addresses can access TiDB Cloud Serverless.  
+    - By default, all IP addresses (`0.0.0.0 - 255.255.255.255`) are allowed.  
+    - You can update allowed IP addresses after cluster creation.  
+- Your cluster has a publicly resolvable DNS name.  
+- Network traffic to and from your cluster is routed over the **public internet** rather than a private network.
+
+### Firewall rules
+
+Granting access to an IP address is done via **firewall rules**. If a connection attempt originates from an unapproved IP address, the client will receive an error.
+
+You can create a maximum of 200 IP firewall rules.
+
+### Allow AWS access
+
+You can enable access from **all AWS IP addresses** by referring to the official [AWS IP address list](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html).  
+
+TiDB Cloud regularly updates this list and uses the reserved IP address **169.254.65.87** to represent all AWS IP addresses.
+
+## Create and manage a firewall rule 
+
+This section provides an overview of managing firewall rules after creating a TiDB Cloud Serverless cluster. With a public endpoint, the connections to the TiDB Cloud Serverless cluster are restricted to allowed IP addresses only. The client IP addresses need to be allowed in firewall rules.
```

Comment:
```suggestion
This section describes how to manage firewall rules for a TiDB Cloud Serverless cluster. With a public endpoint, access to the cluster is restricted to the IP addresses specified in the firewall rules.
```

Reply:
The word "access" has expanded in scope.

Reply:
ok, I've updated "acess" to "the connections to the TiDB Cloud Serverless cluster"